NOTICE.txt

```diff
@@ -2601,6 +2615,19 @@ SOFTWARE.
 
 ---
 
+## regenerator-runtime
+
+This product contains 'regenerator-runtime' by XuCat.
+
+For WeChat using async/await
+
+* HOMEPAGE:
+  * https://github.com/xubaifuCode/regeneratorRuntime
+
+* LICENSE: 
```

Comment:
Doesn't have a license.

Reply:
Looks like they come from here: https://github.com/DefinitelyTyped/DefinitelyTyped

Reply:
It's this one: https://github.com/facebook/regenerator